### ISSUE TITLE
is it possible to build darwin on debian/ubuntu with libboost 1.67?

### ISSUE THREAD

**Issue by aderumier** [VultureProject/darwin#154]
Hi,
I'm currently try to build darwin on debian buster,
but I'm getting this boost error on all plugins build;

```
/root/darwin/darwin-1.2.1/samples/base/Session.cpp:30:54: error: no matching function for call to ‘boost::asio::basic_stream_socket<boost::asio::local::stream_protocol>::basic_stream_socket(<brace-enclosed initializer list>)’
               _cache{cache}, _cache_mutex{cache_mutex} {}
```
currently last debian/ubuntu have libboost 1.67, but the darwin build doc said that 1.70 is needed.

do you think it's related ?   Do you have already build darwin on debian/ubuntu? (I'm seeing some comments in the git log about ubuntu)

**Comment by HugoSoszynski** [comment]
Hi,

This error is due to the initialization of an asio socket with a custom executor.
This feature became available in Boos Asio 1.70.0 ([changelog](https://www.boost.org/users/history/version_1_70_0.html))

`- Added the ability to use custom I/O executors with I/O objects (such as sockets)`

This issue will not be fixed as we do not officially support Linux _(but we try to do our best)_.

I do my tests on Ubuntu 19.10 with a newer Boost version I compiled.


Regards

**Comment by aderumier** [comment]
ok, no problem thanks  !

I'll try to build boost 1.7 myself.
